[2024-11-23T16:08:06.454+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-23T16:08:06.892+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sensor_data_consumer.start manual__2024-11-23T16:08:03.322639+00:00 [queued]>
[2024-11-23T16:08:06.949+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sensor_data_consumer.start manual__2024-11-23T16:08:03.322639+00:00 [queued]>
[2024-11-23T16:08:06.950+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-23T16:08:07.049+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): start> on 2024-11-23 16:08:03.322639+00:00
[2024-11-23T16:08:07.113+0000] {standard_task_runner.py:64} INFO - Started process 728 to run task
[2024-11-23T16:08:07.256+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sensor_data_consumer', 'start', 'manual__2024-11-23T16:08:03.322639+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/***_consumer.py', '--cfg-path', '/tmp/tmpaeii8mb1']
[2024-11-23T16:08:07.267+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask start
[2024-11-23T16:08:37.513+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 7 for task start ((psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8); 728)
[2024-11-23T16:08:52.603+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-11-23T16:08:52.802+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-23T16:08:52.840+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
